app/models/manageiq/providers/nuage/network_manager/vsd_client/rest.rb

```diff
@@ -0,0 +1,79 @@
+require 'rest-client'
+require 'rubygems'
+require 'json'
```

Comment:
~~~These `require`s shouldn't be necessary.~~~

Oh, maybe `rest-client` and `json` should stay, but not `rubygems`.